src/domain/fetchAPI/fetchFromParams.js

```
@@ -0,0 +1,17 @@
+const fetchFromParams = async (url, options) => {
+  try {
+    const response = await fetch(`${url}${options}`);
+
+    if (!response.ok) {
+      throw new Error(`Error ${response.status}`);
+    }
+
+    const result = await response.json();
+    return result;
```

I would suggest to throw errors in case of error, and not silently fail